### PR TITLE
APM-43: Migration Glue name changed to Acquia Platform Migration(APM).

### PR DESCRIPTION
**Pull request by gulabbisht**
https://acquiaps.atlassian.net/browse/APM-43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,15 @@
-# migration_glue
+# acquia_platform_migration
```
Comment (joshirohit100) [inline README.md]: Change this to `Acquia Platform Migration`

File: modules/examples/apm_paragraph_migrate/config/install/migrate_plus.migration.content_paragraph_data_migrate.yml [hunk 2]
```diff
@@ -33,4 +33,4 @@ destination:
 dependencies:
   enforced:
     module:
-    - paragraph_migrate
\ No newline at end of file
+    - apm_paragraph_migrate
```
Comment (joshirohit100) [inline modules/examples/apm_paragraph_migrate/config/install/migrate_plus.migration.content_paragraph_data_migrate.yml]: pls remove white space.

File: modules/examples/apm_paragraph_migrate/config/install/migrate_plus.migration.content_paragraph_migrate.yml [hunk 3]
```diff
@@ -87,4 +87,4 @@ migration_dependencies:
 dependencies:
   enforced:
     module:
-    - paragraph_migrate
\ No newline at end of file
+    - apm_paragraph_migrate
```
Comment (joshirohit100) [inline modules/examples/apm_paragraph_migrate/config/install/migrate_plus.migration.content_paragraph_migrate.yml]: same here

**Comment by joshirohit100** [comment]
Reviewed and there are minor feedbacks.

Also just to confirm, these changes work fine as expected? @gulabbisht

File: modules/examples/apm_paragraph_migrate/config/install/migrate_plus.migration.content_paragraph_data_migrate.yml (hunk 2 above)
Comment (gulabbisht) [reply modules/examples/apm_paragraph_migrate/config/install/migrate_plus.migration.content_paragraph_data_migrate.yml]: I think you mean to add space here.

File: src/Plugin/migrate/process/MGCallbackPlus.php [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Drupal\migration_glue\Plugin\migrate\process;
+namespace Drupal\acquia_platform_migration\Plugin\migrate\process;
```
[the remark this answers is not in the document]
Comment (gulabbisht) [reply src/Plugin/migrate/process/MGCallbackPlus.php]: Update and made related changes as well.

File: src/Plugin/migrate/process/MigrationGlueTermHierarchy.php [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Drupal\migration_glue\Plugin\migrate\process;
+namespace Drupal\acquia_platform_migration\Plugin\migrate\process;
```
[the remark this answers is not in the document]
Comment (gulabbisht) [reply src/Plugin/migrate/process/MigrationGlueTermHierarchy.php]: Update and made related changes.

**Pull request merged**